mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosFineGrainedSchedulerBackendSuite.scala

```diff
@@ -345,7 +374,17 @@ class MesosFineGrainedSchedulerBackendSuite
       2 // Deducting 1 for executor
     )
 
-    val taskDesc = new TaskDescription(1L, 0, "s1", "n1", 0, ByteBuffer.wrap(new Array[Byte](0)))
+    val taskMetrics = TaskMetrics.empty
+    val taskContextImpl = mock[org.apache.spark.TaskContextImpl]
+    val task = new org.apache.spark.scheduler.Task[Int](0, 0, 0) {
+      context = taskContextImpl
+      taskMetrics.incMemoryBytesSpilled(10)
+      override def runTask(tc: TaskContext): Int = 0
+    }
+    val taskDesc = new TaskDescription(1L, 0, "s1", "n1", 0,
```

Comment:
nit: with so many args, all with pretty generic types, its really helpful to name each one, eg
```
TaskDescription(
  taskId=1L,
  attemptNumber = 0,
...
```

(here and elsewhere)